tests/baselines/reference/defineVariables_useDefineForClassFields.types

```diff
@@ -0,0 +1,16 @@
+=== tests/cases/compiler/defineVariables_useDefineForClassFields.ts ===
+const a = () => b()
+>a : () => any
+>() => b() : () => any
+>b() : any
+>b : () => any
+
+const b = () => null
+>b : () => any
+>() => null : () => any
+>null : null
+
+a()
+>a() : any
+>a : () => any

```

Comment:
Interesting that this is the result type; I guess because `--strictNullChecks` is off by default in tests?

Might be better to change from `() => null` to `() => 0` just to make sure the types propagate properly (or enable `--strict`)

Reply:
I don't think types matter that much for this test since it's just trying to show the absence of an error.